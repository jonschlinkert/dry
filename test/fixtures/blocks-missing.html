{% extends "missing-block" %}
{% block "one" %}
one: "block-missing.html"
{% endblock %}
{% block "two" %}
two: "block-missing.html"
{% endblock %}
{% block "three" %}
THIS BLOCK SHOULD NOT RENDER
{% endblock %}