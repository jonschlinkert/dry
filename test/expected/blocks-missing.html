
one: "block-missing.html"

two: "block-missing.html"
